Swift-Base/AppDelegate.swift

```diff
@@ -106,3 +76,69 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 }
 
+extension AppDelegate {
+    
+    func setupProject() {
```

Comment:
I think ``setupProject`` code must be in ``didFinishLaunchingWithOptions`` because the last not contains anything else, but :+1: